tests/integration/targets/lookup_passwordstore/tasks/tests.yml

```diff
@@ -61,6 +61,37 @@
     that:
       - readpass == newpass
 
+- name: Create a password using missing=create
+  set_fact:
+    newpass: "{{ lookup('community.general.passwordstore', 'test-missing-create missing=create length=8') }}"
+
+- name: Fetch password from an existing file
+  set_fact:
+    readpass: "{{ lookup('community.general.passwordstore', 'test-missing-create') }}"
+
+- name: Verify password
+  assert:
+    that:
+      - readpass == newpass
+
+- name: Fetch password from existing file using missing=empty
+  set_fact:
+    readpass: "{{ lookup('community.general.passwordstore', 'test-missing-create missing=empty') }}"
+
+- name: Verify password
+  assert:
+    that:
+      - readpass == newpass
+
+- name: Fetch password from non-existing file using missing=empty
+  set_fact:
+    readpass: "{{ lookup('community.general.passwordstore', 'test-missing-pass missing=empty') }}"
+
+- name: Verify password
+  assert:
+    that:
+      - readpass == []
```

Comment:
Shouldn't `missing=empty` return an empty string? If not, the documentation suggestion I added needs to be updated to clarify what `missing=empty` actually does.

Reply:
It actually always returns a list of strings, usually one. If I understood the code correctly, it might also be more in case of password update.

Reply:
There's always one password added for every term, so multiple entries can only happen if you specify multiple terms. This has been true only until now though, and I'm still not sure whether the new state - empty list instead of a one-element list (which is converted to its element by `lookup`, as opposed to when `query` is used) is a good idea. It might be better to return a one-element list whose element is an empty string or `None`.

Reply:
No problem, if you think it is a better idea, I can make sure there will be at least `None` in that list so we really get an empty string in that case.